### ISSUE TITLE
Compilation on 64bit PIC

### ISSUE THREAD

**Issue by plata** [shogun-toolbox/shogun#2822]
I am not able to compile shogun on my 64bit openSuse. I had to add `COMPILE_FLAGS "-fPIC"` to the properties of libshogun (src/shogun/CMakeLists.txt) to make it work.

I think this should be an issue on every 64bit machine. Solve by making libshogun PIC or build shogun and shogun-static directly from the sources (without libshogun).

**Comment by lisitsyn** [comment]
Thanks! Going to commit a fix in a few minutes

**Comment by besser82** [comment]
See my comment in https://github.com/shogun-toolbox/shogun/commit/caf0871a8223ae71a6003e84fca14e0140fdfe53

`-fPIC` is already enabled for libshogun.  Can you please paste a log of the cmake-output and build-log of terminal on fpaste.org?  Don't forget to add `DCMAKE_VERBOSE_MAKEFILE:BOOL=ON` to your cmake-invocation…

**Comment by plata** [comment]
I compiled shogun 4.0.0 (from http://www.shogun-toolbox.org/) and 3.1.1 (as according to the wiki) because I wanted a stable solution. Did not see that it was enabled on Github already, so sorry for that. 
I also tried to POSITION_INDEPENDENT_CODE but it didn't work. I can checkout the current Github sources tomorrow. Not sure if this policy http://www.cmake.org/cmake/help/v3.0/policy/CMP0018.html might cause some trouble.
